README.md

```diff
@@ -10,21 +10,43 @@ You can see https://oxc-project.github.io/blog/2023-12-12-announcing-oxlint.html
 
 ```shell
 pnpm add eslint-plugin-oxlint --D
+# or `npm i -D eslint-plugin-oxlint`
+# or `yarn add -D eslint-plugin-oxlint``
 ```
 
 ## Usage
 
-This is a flat config usage.
+### Flat config
+
+This plugin is optimized for flat config usage (eslint >= 9.0). See [here](https://eslint.org/docs/latest/use/configure/configuration-files-new) for more details. Use it like this:
 
 ```js
 // eslint.config.js
-import oxlint from "eslint-plugin-oxlint"
+import oxlint from "eslint-plugin-oxlint";
 export default [
-  ... // other plugins
+  ...// other plugins
   oxlint, // oxlint should be the last one
-]
+];
+```
+
+### Legacy config
+
+If you are using legacy configuration (eslint < 9.0), you can use the following config:
+
+```js
+// .eslintrc.js
+const { oxlintRules } = require('eslint-plugin-oxlint')
+module.exports = {
+  ... // other config
+  rules: {
+    ... // other rules
+    ...oxlintRules // oxlint rules should be the last ones
```

Comment:
I think we should support passing ' plugins: [oxlint]` when the eslint version < 9.0. But I haven't looked into what needs to be done.

Reply:
I agree with you. In the meantime, doing like this works well and is not too hacky to setup, imho.